docs/en_US/NniOnWindows.md

```diff
@@ -23,9 +23,11 @@ Set-ExecutionPolicy -ExecutionPolicy Unrestricted
 * __Install NNI through source code__
```

Comment:
It looks duplicated with homepage, you can add a link here only.

Reply:
have fixed